core/block.js

```diff
@@ -206,6 +221,15 @@ Blockly.Block = function(workspace, prototypeName, opt_id) {
   }
 };
 
+/**
+ * @typedef {{
+ *            text:?string,
+ *            pinned:boolean
```

Comment:
You just need a comma after ``pinned:boolean`` to fix typings ;)

Reply:
Oh gosh (－‸ლ) thank you for the help.